src/views/LibraryPlayer/LibraryPlayer.tsx

```diff
@@ -69,9 +74,27 @@ class _LibraryPlayer extends React.PureComponent<LibraryPlayerProperties, Librar
             games_checked: {},
             new_collection_name: "",
             new_collection_private: false,
+            sort_order: "date_added",
+            sort_descending: true,
```

Comment:
I reckon it'd be nice if we remembered these, that's usually done with the preferences system, which goes something like this

1) Add your preference keys and default values to `lib/preferences.ts", so something like `"sgf.sort-order": "date_added"`
2) `import * as  preferences from "preferences";`
3) 
```
sort_order: preferences.get("sgf.sort-order"),
sort_descending: preferences.get("sgf.sort-descending"),
```
4) When you change your sort order, call `preferences.set("sgf.sort-order", whatever)`

Reply:
Will do!